### PR TITLE
Update faces

### PR DESCRIPTION
**Pull request by mirisuzanne**
## Cute animal pic
_Because everyone likes pictures of [animals](https://unsplash.com/t/animals)._

## Link to Trello card (if applicable)
_If this PR relates to a Trello card, don't forget to reference this PR on the card as well._

https://trello.com/c/D7aSlxcr/111-move-kit-to-emeritus-add-new-birds

## Description
_The commit messages say what you did; this should explain why and/or how._
- [x] Description is in Trello card

## Steps to test/reproduce
_Please explain how to best reproduce the issue and/or test the changes locally (including the pages/URLs/views/states to review)._

Check out the about/bird pages!

### CODE REVIEW

**Comment by mirisuzanne** [comment]
[Move Kit to Emeritus & Add New Birds](https://trello.com/c/D7aSlxcr/111-move-kit-to-emeritus-add-new-birds)

File: content/birds/patricia.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+title: Patricia Realini
+sub: Technical Project Manager
+bird: patricia
+date: 2020-11-02
+image:
```
Comment (mirisuzanne) [inline content/birds/patricia.md:6]: @patriciarealini you can check the [deploy preview](https://5fb6f77ee3041900071ca631--oddleventy.netlify.app/authors/patricia/) or run this branch locally and go to `localhost:8080/authors/patricia/` to see how the image is sized/aligned. It looks like it needs a bit of adjustment. A few notes on that:

- Size: the `height` and `width` properties here should match the actual pixel size of the source image. That's used to give it an intrinsic ratio, and also to generate appropriate smaller versions for `srcset`.
- Alignment: you can crop the source as desired, or use a `position` property to set the CSS `object-position`, which defaults to `center`. (it looks to me like you might want to crop some, and then also adjust the position)

**Review by mirisuzanne — COMMENTED** [review]
@jerivas @patriciarealini @dvdherron I merged all your branches in here, and added the faces. Take a look and make any final adjustments you need.

**Comment by jerivas** [comment]
@mirisuzanne for some reason all faces appear stacked on the preview link 
image

**Comment by mirisuzanne** [comment]
@jerivas oops, I guess that approach doesn't work on all the browsers… back to the drawing board!

**Comment by dvdherron** [comment]
Besides the issues with Patricia's photo this looks good to me. the drawings look great! thanks for that.

**Review by dvdherron — APPROVED** [review]

**Review by jgerigmeyer — APPROVED** [review]

File: content/birds/miriam.md [hunk 2]
```diff
@@ -25,27 +25,27 @@ social:
   stackoverflow: 1930386
 summary: |
   Miriam is a co-founder and front-end architect --
-  helping oversee the user-experience
+  overseeing the user-experience
```
Comment (jgerigmeyer) [inline content/birds/miriam.md:28]: @mirisuzanne I made this change so that your and my bios don't both start with `___ is a ___ -- helping ___`. 😄

Comment (mirisuzanne) [reply content/birds/miriam.md:28]: ha, fine - we can be _slightly_ different people

File: content/birds/patricia.md (hunk 1 above)
Comment (patriciarealini) [reply content/birds/patricia.md:6]: thanks for the details and breakdown, i ended up following the suggestions in your second bullet point. let me know if there are any further clean ups i need to do.

Comment (mirisuzanne) [reply content/birds/patricia.md:6]: looks great to me. 👍

**Pull request merged**